src/data/dem_data.js

```diff
@@ -95,6 +87,26 @@ class DEMData {
         this.loaded = true;
     }
 
+    unpackMapbox(r: number, g: number, b: number) {
```

Comment:
I think we can use `_unpack*` for these functions to signify that they're private to the `DEMData` class

Reply:
Done!